### PR TITLE
add munichjs announcement

### PR DESCRIPTION
**Pull request by marcoow**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -40,14 +40,14 @@
...
+            <p><a href="https://glimmerjs.com">Glimmer.js</a> can be used to build <strong>native web components</strong> for reuse in any frontend stack. This article goes deep into how and why to do that.</p>
             <a href="https://simplabs.com/blog/2017/08/28/creating-web-components-with-glimmer.html" class="custom-button">learn more</a>
           </div>
         </div>
         <div class="col-sm-6 content-teaser">
           <div>
-            <h4>Meet us at iJS in Munich</h4>
-            <p>The international JavaScript Conference will take place in Munich, Oct. 23rd to 27th. <strong>Be sure to stop by our booth</strong> for info on Ember.js and Glimmer (and some swag)!</p>
+            <h4>Feel the Glimmer at MunichJS</h4>
+            <p><a href="https://twitter.com/marcoow">@marcoow</a> will be speaking about the <strong>internals of Ember's Glimmer VM, <a href="https://glimmerjs.com">Glimmer.js</a></strong> and its role in the <strong>future of <a href="https://emberjs.com">Ember.js</a></strong> at the next <a href="https://www.meetup.com/de-DE/preview/MunichJS-User-Group/events/244480513">MunichJS</a> at Nov 2nd.</p>
```
Comment (jayjayjpg) [inline index.html]: I think as `at` stands with times and `on` with dates, this should be
```
[...] at the next <a href="https://www.meetup.com/de-DE/preview/MunichJS-User-Group/events/244480513">MunichJS</a> on Nov 2nd.</p>
```
from [English Stack Exchange](https://english.stackexchange.com/questions/182660/on-vs-at-with-date-and-time)

**Review by jayjayjpg — COMMENTED** [review]
This looks good, just a minor grammar comment from my side 👍

File: index.html (hunk 1 above)
Comment (marcoow) [reply index.html]: absolutely 👍

Comment (marcoow) [reply index.html]: fixed

**Review by jayjayjpg — APPROVED** [review]

**Pull request merged**